jedi/_compatibility.py

```diff
@@ -18,6 +19,19 @@
 is_py26 = not is_py3 and sys.version_info[1] < 7
 
 
+class DummyFile(object):
```

Comment:
No, that's fine. There's also import caching that won't let you import the same thing over and over again.